src/lib/journey/callbacks/text-output/text-output.mock.ts

```diff
@@ -21,11 +21,11 @@ export default {
       output: [
         {
           name: 'message',
-          value: 'Have you read our <a href="#">privacy policy</a>?',
+          value: 'There is been a login attempt in your account, please check!',
         },
         {
           name: 'messageType',
-          value: '0',
+          value: '1',
```

Comment:
as a note to self - this makes me think we should create a union in the sdk for this callbacks message types. right now its string and i think we can do a much better job of hinting to the developer whats happening. we can make a mapped type that probably describes what it is too.